ci/build_wheel.sh

```diff
@@ -16,10 +16,26 @@ rapids-generate-version > ./VERSION
 
 cd "${package_dir}"
 
+case "${RAPIDS_CUDA_VERSION}" in
+  12.*)
+    EXCLUDE_ARGS=(
+      --exclude "libcublas.so.12"
+      --exclude "libcublasLt.so.12"
+      --exclude "libcurand.so.10"
```

Comment:
I don't believe cuvs depends on curand at all.

Reply:
See `cpp/CMakeLists.txt` - it links `cuvs` against `CUDA::curand`

Reply:
I don't see any use of cuRAND headers. Let's try to remove the curand dependency in a follow-up PR.

Reply:
raft uses curand via the `raft/random` headers, so we bring in the dependency on curand symbols via that.

When we look at the symbols for `libcuvs.so` in the 24.10 nightly wheels we see curand symbols embedded in it:
```
nm -D libcuvs.so | grep curand
.....
00000000040aad60 T curandCreateGenerator
00000000040a7030 T curandCreateGeneratorHost
00000000040a88f0 T curandCreatePoissonDistribution
00000000040a8870 T curandDestroyDistribution
00000000040a7ad0 T curandDestroyGenerator
00000000040b5b60 T curandGeneratePoissonMethod
00000000040abcb0 T curandGenerateSeeds
00000000040c5360 T curandGenerateUniform
00000000040ca160 T curandGenerateUniformDouble
00000000040a8af0 T curandGetDirectionVectors32
00000000040a8f00 T curandGetDirectionVectors64
00000000040a7750 T curandGetProperty
00000000040a8f40 T curandGetScrambleConstants32
00000000040a90d0 T curandGetScrambleConstants64
00000000040a7740 T curandGetVersion
00000000040a7810 T curandSetGeneratorOffset
00000000040a78a0 T curandSetGeneratorOrdering
00000000040a77d0 T curandSetPseudoRandomGeneratorSeed
00000000040a7a00 T curandSetQuasiRandomGeneratorDimensions
00000000040a7790 T curandSetStream
0000000007780660 R curand_precomputed_mt19937_polynomials_1
0000000008140660 R curand_precomputed_mt19937_polynomials_2
```

So yes the dependency on curand is required

Reply:
Ugh you are right. We have been trying to get rid of the curand dependency even if raft but we aren’t there quite yet.